test/extensions/filters/network/common/fuzz/network_readfilter_fuzz.proto

```diff
@@ -27,6 +27,10 @@ message Action {
   }
 }
 
+message FuzzHelperForActions {
```

Comment:
Please add a small comment why this message is needed.

Reply:
Added.